lib/PuppeteerSharp/Page.cs

```diff
@@ -463,7 +469,7 @@ public Task<JSHandle> QuerySelectorAllHandleAsync(string selector)
         }
 
         /// <summary>
-        /// Adds a <c><script></c> tag into the page with the desired url or content
+        /// Adds a <c><![CDATA[<script>]]></c> tag into the page with the desired url or content
```

Comment:
is this needed?

Reply:
Yes. `<script>` breaks the XML so it would be ignored by the compiler.